tests/lava/magma/core/process/test_ports.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (C) 2021 Intel Corporation
 # SPDX-License-Identifier:  BSD-3-Clause
```

Comment:
One of those old files with two spaces. Fix these whenever you see these.

Reply:
Done